core/components/commerce_timeslots/src/Modules/TimeSlots.php

```diff
@@ -63,11 +63,9 @@ public function initialize(EventDispatcher $dispatcher)
 
 
         // Requires the Scheduler service in Commerce 1.3+
-        $root = $this->commerce->config['core_path'];
-        $path = $root . 'src/Services/Scheduler/crontime';
-        if (file_exists($path)) {
+        if ($this->commerce->isSchedulerActive()) {
```

Comment:
Maybe this could also schedule it if the scheduler isn't active (perhaps to allow a different scheduler implementation that doesn't write the crontime file), but it does need to check if the scheduler exists (i.e pre-1.3). 

Or.... we restrict the upgrade to 1.3+ so we're sure it'll be on 1.3. Could still have an older version available for on 1.1+...

Reply:
We could also hide all mention of the repeat scheduling if it's under 1.3
That way it would be immediately available on upgrading.

The way it currently is, the repeat schedules can be set, but they won't actually run until Commerce gets upgraded.